doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -153,6 +153,36 @@ Now, we have a DataFrame containing only two columns, "features" which contains
 "sepal length", "sepal width", "petal length" and "petal width" and "classIndex" which has Double-typed
 labels. A DataFrame like this (containing vector-represented features and numeric labels) can be fed to XGBoost4J-Spark's training engine directly.
 
+Dealing with missing values (NaN / Null):
+--------------------------
+
+In case a feature (column) contains missing values (NaN / NULLs) for any reason (could be related to business logic / wrong data ingestion process / any other reason), one should handle it in the below manner, following the below steps:
```

Comment:
it's not true actually, if the feature is `NaN` instead of non-exist or as NULL, 

users should choose to ignore it in vector assembler or keep it to let xgboost handle

Reply:
@CodingCat note I've fixed it and added it to the document.

Reply:
@CodingCat per below note, in case it's NaN - I will refer the user to the below (new text, still not committed) section. In case it's Null - I will keep the above text.

WDYT?